typings/index.d.ts

```diff
@@ -72,4 +72,9 @@ declare module 'prismarine-nbt'{
   export function comp<T extends object> (val: T): { type: 'compound', value: T }
   export function int<T extends number> (val: T): { type: 'int', value: T }
   export function list<T extends string, K extends {type: T}>(...value: K[]): { type: 'list'; value: { type: T | 'end', value: K[] } };
+  export function double<T extends number> (value: T): { type: 'double', value: T}
+  /**
+   * @param values Javascript doesn't support 64-bit integers,so this method takes an array of two 32-bit integers
+   */
+  export function long<T extends number> (values: T[]): { type: 'long', value: T[]}
```

Comment:
You can also pass BigInts to protodef. For 64-bit values, you can get a BigInt when reading with .valueOf()

Reply:
Good to know, thanks.